pkg/kube/template.go

```diff
@@ -43,13 +44,23 @@ func NewTemplate(template map[string]interface{}, backend types.Backend, prefix
 		kvVersion = kv
 	}
 
-	// Only worry about getting Vault secrets for templates with <placeholder>'s
-	replaceable := replaceableInner(&template)
-	var data map[string]interface{}
-	if replaceable {
-		data, err = backend.GetSecrets(path, kvVersion)
+	var avpIgnore bool
+	if ignore, ok := annotations["avp_ignore"]; ok {
+		avpIgnore, err = strconv.ParseBool(ignore)
 		if err != nil {
-			return nil, err
+			avpIgnore = false
+		}
+	}
+
+	var data map[string]interface{}
+	var replaceable bool
+	if !avpIgnore {
+		replaceable = replaceableInner(&template)
```

Comment:
Instead of doing
```
var replaceable bool
if !avpIgnore {
  replaceable = replaceableInner(&template)
  if replaceable {
```
You could do
```
if replaceableInner(&template)
```

Reply:
I need to store the result of `replaceableInner`  for line 71 where we return the constructed `Template`

Reply:
And would prefer to not run `replaceableInner` twice since right now it runs through the whole file looking for `<thing>`'s

Reply:
Sure, should have looked further down. It just looks a little funky with the 3 different handlings of the bool, not a blocker